packages/fxa-graphql-api/src/gql/lib/error.ts

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ApolloError } from 'apollo-server';
+
+export const PROFILE_INFO_URL =
+  'https://github.com/mozilla/fxa/blob/main/packages/fxa-profile-server/docs/API.md#errors';
+
+export function CatchGatewayError(
+  target: any,
+  propertyKey: string,
+  descriptor: PropertyDescriptor
+) {
+  const originalMethod = descriptor.value;
+  descriptor.value = async function (...args: any[]) {
+    try {
+      return await originalMethod.apply(this, args);
+    } catch (err) {
+      if (err.code && err.errno && err.message) {
```

Comment:
Kind of stinks that auth-server and profile-server return different error objects and this is the only way to tell them apart.

Is there something in graphql that might make this better?